cmd/private_network_create.go

```diff
@@ -22,11 +20,15 @@ type privateNetworkCreateCmd struct {
 
 	Name string `cli-arg:"#"`
 
-	Description string `cli-usage:"Private Network description"`
-	EndIP       string `cli-usage:"managed Private Network range end IP address"`
-	Netmask     string `cli-usage:"managed Private Network netmask"`
-	StartIP     string `cli-usage:"managed Private Network range start IP address"`
-	Zone        string `cli-short:"z" cli-usage:"Private Network zone"`
+	Description  string      `cli-usage:"Private Network description"`
+	EndIP        string      `cli-usage:"managed Private Network range end IP address"`
+	StartIP      string      `cli-usage:"managed Private Network range start IP address"`
+	Zone         v3.ZoneName `cli-short:"z" cli-usage:"Private Network zone"`
+	Netmask      string      `cli-usage:"managed Private Network netmask"`
+	DNSServers   []string    `cli-usage:"DNS servers"`
+	NTPServers   []string    `cli-usage:"NTP servers"`
+	Routers      []string    `cli-usage:"Routers"`
+	DomainSearch []string    `cli-usage:"Domain search list, limited to 255 octets"`
```

Comment:
```suggestion
	DNSServers   []string    `cli-flag:"dns-server" cli-usage:"DNS servers"`
	NTPServers   []string    `cli-flag:"ntp-server" cli-usage:"NTP servers"`
	Routers      []string    `cli-flag:"router" cli-usage:"Routers"`
	DomainSearch []string    `cli-usage:"Domain search list, limited to 255 octets"`
```

Make flag singular (var remains plural).

Reply:
Since you specify one flag per item it should be singular (eg. `--dns-server 1.1.1.1`).
We already do the same thing for flags such as `--ssh-key` (see [here](https://github.com/exoscale/cli/blob/master/cmd/instance_create.go#L43))